lib/qless/server/static/css/style.css

```diff
@@ -134,6 +134,20 @@ body { padding:0; margin:0; }
 .subnav li.current a { background:#fff; font-weight:bold; color:#ce1212;}
 .subnav li a:active { background:#b00909;}
 
+.subnav.subnav-fixed.page-layout {
```

Comment:
Feels like `.page-layout` should be its own element. Is this class completely related to `.subnav`? If not I would suggest adding `.page-layout` as an external `div` instead of having it with `.subnav .subnav-fixed`.

Reply:
@mateus maybe `page-layout` isn't a good name, but I want to override some of the styles applied by `subnav` and creating an external div doesn't seem to do this.

Reply:
If it's modifying `.subnav` you can use `.subnav--modifier-name` as your class name. 

We use at Shopify a methodology called BEM (Block Element Modifier). You can have a better understanding by reading this http://getbem.com/introduction/

Long story short we have 3 possible elements. A Block, an Element and a Modifier:

- `Block` is the main node. With all the default styles. (e.g.`.subnav`)

- `Element` is something inside the main block that has a direct relation with it. (e.g. `.subnav__item`, separated with two underscores) 

- `Modifier` is what modifies the default styles of a `Block` or `Element`. (e.g. `.subnav__item--active`, separated with two dashes)